test/Microsoft.Health.SqlServer.Web/Microsoft.Health.SqlServer.Web.csproj

```diff
@@ -31,12 +36,12 @@
     <EmbeddedResource Include="Features\Schema\Migrations\3.sql" />
     <GenerateFilesInputs Include="Features\Schema\Migrations\2.sql" />
     <GenerateFilesInputs Include="Features\Schema\Migrations\3.sql" />
-    <Generated Include="Features\Schema\Model\VLatest.Generated.cs">
+    <Generated Include="Features\Schema\Model\VLatest.Generated.$(TargetFramework).cs">
```

Comment:
Was there any specific need to this way? Did something broke with having older framework file?

Reply:
This is to prevent issues with concurrent builds for multiple build targets. In truth, this code should be generated using source generators instead.